client/src/pages/AddReview/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+// https://www.npmjs.com/package/react-typeform-embed
+import { ReactTypeformEmbed } from 'react-typeform-embed';
+
+import { Button } from '../../components';
+import urls from './urls';
+
+import './style.css';
+
+let typeformEmbed = null;
+const openForm = () => typeformEmbed.typeform.open();
+
+const AddReview = ({ type, sitspotId }) => (
+  <>
+    <div className="header-img header-container" />
+    <div className="add-new-review-container">
+      <h1 className="add-new-review__title">
+        We will now ask you to follow these steps to complete your
+        recommendation:
+      </h1>
+      <ol>
+        <li>
+          1) Fill in your contributor profile - only if you haven&apos;t already
```

Comment:
`ol` should give you a numbered list, the `1)` shouldn't be necessary? (same below)

Reply:
and for some reason the ol, ul list styles are not showing :(

Reply:
for style reasons cant use ul bc they wont show the list-style next to our text

Reply:
that looks like a css bug that should be fixed.. you can do a quick search on your css files to see if you override styles for ul/ol/li somewhere, or use browser's dev tools to look at the element and its styles
https://stackoverflow.com/questions/2642067/ordered-list-ol-showing-up-un-numbered
Also, if you wanna show the list in certain way, use css, don't just work around it by using div and br - ul/ol exists for a reason, one of which is for example being able to be processed and navigated by a screen reader for visually impaired people